site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -221,13 +221,11 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition                                                                                                    | `false`                            |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                                                                                                                                                     | `false`                            |
 
-{{% feature publishVersion="1.29.0" %}}
```

Comment:
Rather than deleting this line, make this `publishVersion` 1.30.0 instead.

This is how feature shortcode's work: https://agones.dev/site/docs/contribute/#within-a-page